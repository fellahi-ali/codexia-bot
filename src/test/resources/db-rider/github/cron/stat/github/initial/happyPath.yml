github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 61200117
        full_name: instaloader/instaloader
        html_url: https://github.com/instaloader/instaloader
        repo_created_at: 2019-08-14 04:42:36

github_repo_source:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "2222"

github_repo_stat: []
