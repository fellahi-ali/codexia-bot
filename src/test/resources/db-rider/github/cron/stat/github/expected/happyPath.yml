github_repo_stat:
    -
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"description":"Download pictures (or videos) along with their captions and other metadata from Instagram.","pushedAt":[2020,5,11,18,27,23],"language":"Python","homepage":"https://instaloader.github.io/","forks":281,"watchers":1561,"stars":1561,"type":"GITHUB_API"}'
